skyreader/utils/handle_map_data.py

```diff
@@ -101,14 +112,21 @@ def extract_map(
         # show 2 * delta_LLH
         grid_value = grid_value * 2.
         equatorial_map *= 2.
+
+        # nan values are a problem for the convolution and the contours
+        max_map = np.nanmax(equatorial_map)
+        equatorial_map[np.isnan(equatorial_map)] = max_map
+        grid_value[np.isnan(grid_value)] = max_map
```

Comment:
The comment there is indeed wrong. It is the result of a bad copy and paste. I did create that because I noticed that nan values can be sometimes a problem when drawing the contours also for llh map. Anyway, it still makes sense to save nan values as well, therefore I modified the code and now the nan values are removed only when determining the contours, otherwise they are still always present.